cpp/include/cudf/io/parquet.hpp

```diff
@@ -1380,8 +1382,9 @@ class parquet_chunked_writer {
   /**
    * @brief Default constructor, this should never be used.
    *        This is added just to satisfy cython.
+   *        This is added to not leak detail API
```

Comment:
Do we need a similar comment on `chunked_parquet_reader`?

Reply:
Yes, added.